### DISCUSSION TITLE
Ability to control visibility of PasswordInput via props

### DISCUSSION THREAD

**Discussion by karandwivedi42** [mantinedev/mantine#2408] [Feature requests]
I want to make a form where the "New Password" and "Confirm Password" inputs toggle their visibility together when the toggle button on any of them is clicked.

**Comment by rtivital** [comment]
Hi @karandwivedi42, thanks for a feature request, it will be added in one of the next releases

**Comment by karandwivedi42** [comment]
Thanks! 

https://mantine.dev/core/password-input/#controlled-visibility-toggle
